test/debug/quit_test.rb

```diff
@@ -27,6 +27,13 @@ def test_quit_does_not_quit_debugger_process_if_not_confirmed
       end
     end
 
+    def test_quit_quits_immediately_if_no_confirm_quit_is_set_to_true
+      debug_code(program) do
+        type 'config set no_confirm_quit true'
+        type 'q'
+      end
+    end
```

Comment:
This spec doesn't pass. I suspect it's because of `test/tool/test_builder.rb:121`, but I'm not sure. If the maintainers are interested in this PR I'll happily spend some time trying to make this spec pass.